docs/resources/cluster_v2.md

```diff
@@ -232,16 +232,30 @@ resource "rancher2_machine_config_v2" "foo-harvester-v2" {
     vm_namespace = "default"
     cpu_count = "2"
     memory_size = "4"
-    disk_size = "40"
-    network_name = "harvester-public/vlan1"
-    image_name = "harvester-public/image-57hzg"
+    disk_info = <<EOF
+    {
+        "disks": [{
+            "imageName": "harvester-public/image-57hzg",
+            "size": 40,
+            "bootOrder": 1
+        }]
+    }
+    EOF
+    network_info = <<EOF
+    {
+        "interfaces": [{
+            "networkName": "harvester-public/vlan1",
+            "macAddress": ""
+        }]
+    }
+    EOF
     ssh_user = "ubuntu"
   }
 }
 
 resource "rancher2_cluster_v2" "foo-harvester-v2" {
   name = "foo-harvester-v2"
-  kubernetes_version = "v1.22.6+rke2r1"
+  kubernetes_version = "v1.24.9+rke2r2"
```

Comment:
should we update these to the latest in KDM (1.25.6+rke2r1)?

Reply:
@slickwarren Thanks, changed the kubernetes_version to `v1.24.10+rke2r1` which both supported in Rancher v2.6.11 and Rancher v2.7-head.